Description: Create persistdir for testing suite.
Author: Mihai Moldovan <ionic@ionic.de>

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -38,3 +38,5 @@ CONFIGURE_FILE(${CMAKE_SOURCE_DIR}/tests
 CONFIGURE_FILE(${CMAKE_SOURCE_DIR}/tests/modules/etc/dnf/dnf.conf.in ${CMAKE_SOURCE_DIR}/tests/modules/etc/dnf/dnf.conf @ONLY)
 
 CONFIGURE_FILE(${CMAKE_SOURCE_DIR}/etc/dnf/dnf_test.conf.in ${CMAKE_SOURCE_DIR}/etc/dnf/dnf_test.conf @ONLY)
+
+FILE(MAKE_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/tests/dnfpersist)
