Description: Substitute variables in testing configuration file.
Author: Mihai Moldovan <ionic@ionic.de>

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -36,3 +36,5 @@ CONFIGURE_FILE(${CMAKE_SOURCE_DIR}/dnf/c
 CONFIGURE_FILE(${CMAKE_SOURCE_DIR}/dnf/cli/completion_helper.py.in ${CMAKE_SOURCE_DIR}/dnf/cli/completion_helper.py @ONLY)
 CONFIGURE_FILE(${CMAKE_SOURCE_DIR}/tests/modules/etc/dnf/repos.d/test.repo.in ${CMAKE_SOURCE_DIR}/tests/modules/etc/dnf/repos.d/test.repo @ONLY)
 CONFIGURE_FILE(${CMAKE_SOURCE_DIR}/tests/modules/etc/dnf/dnf.conf.in ${CMAKE_SOURCE_DIR}/tests/modules/etc/dnf/dnf.conf @ONLY)
+
+CONFIGURE_FILE(${CMAKE_SOURCE_DIR}/etc/dnf/dnf_test.conf.in ${CMAKE_SOURCE_DIR}/etc/dnf/dnf_test.conf @ONLY)
