From bc69ed55f2ff0dbd36349e5bd5030b142a544cc5 Mon Sep 17 00:00:00 2001
From: Mihai Moldovan <ionic@ionic.de>
Date: Sat, 10 Oct 2020 11:49:55 +0200
Subject: [PATCH] Substitute variables in testing configuration file.

---
 CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5f44c349..b25428e7 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -36,4 +36,6 @@ CONFIGURE_FILE(${CMAKE_SOURCE_DIR}/dnf/const.py.in ${CMAKE_SOURCE_DIR}/dnf/const
 CONFIGURE_FILE(${CMAKE_SOURCE_DIR}/dnf/cli/completion_helper.py.in ${CMAKE_SOURCE_DIR}/dnf/cli/completion_helper.py @ONLY)
 CONFIGURE_FILE(${CMAKE_SOURCE_DIR}/tests/modules/etc/dnf/repos.d/test.repo.in ${CMAKE_SOURCE_DIR}/tests/modules/etc/dnf/repos.d/test.repo @ONLY)
 CONFIGURE_FILE(${CMAKE_SOURCE_DIR}/tests/modules/etc/dnf/dnf.conf.in ${CMAKE_SOURCE_DIR}/tests/modules/etc/dnf/dnf.conf @ONLY)
+
+CONFIGURE_FILE(${CMAKE_SOURCE_DIR}/etc/dnf/dnf_test.conf.in ${CMAKE_SOURCE_DIR}/etc/dnf/dnf_test.conf @ONLY)
 CONFIGURE_FILE(${CMAKE_SOURCE_DIR}/doc/conf.py.in ${CMAKE_SOURCE_DIR}/doc/conf.py @ONLY)
-- 
2.26.2

