From 7d11113995f7c69bee3a298366dbfa11a0d8cf7a Mon Sep 17 00:00:00 2001
From: Mihai Moldovan <ionic@ionic.de>
Date: Sat, 10 Oct 2020 11:49:55 +0200
Subject: [PATCH] Use custom testing dnf configuaration file for (some) tests.

---
 tests/test_cli.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/test_cli.py b/tests/test_cli.py
index 9c130c36..a080eeb8 100644
--- a/tests/test_cli.py
+++ b/tests/test_cli.py
@@ -183,7 +183,7 @@ class ConfigureTest(tests.support.DnfBaseTestCase):
         self.base._plugins = mock.Mock()
         self.cli = dnf.cli.cli.Cli(self.base)
         self.cli.command = mock.Mock()
-        self.conffile = os.path.join(tests.support.dnf_toplevel(), "etc/dnf/dnf.conf")
+        self.conffile = os.path.join(tests.support.dnf_toplevel(), "etc/dnf/dnf_test.conf")
 
     @mock.patch('dnf.util.am_i_root', lambda: False)
     def test_configure_user(self):
-- 
2.26.2

